src/epcpm/importexportdialog.py

```diff
@@ -233,26 +251,34 @@ def pick_tables_c(self):
             multiple=False,
         )
 
-    def pick_sunspec_tables_c(self):
+    def pick_bitfields_c(self):
         filters = (
-            ("SunSpec Tables C", ["c"]),
+            ("Bitfields C", ["c"]),
             all_files_filter,
         )
 
         self.file_dialog(
-            target=self.ui.sunspec_tables_c,
+            target=self.ui.bitfields_c,
             filters=filters,
             multiple=False,
         )
 
-    def pick_sunspec_bitfields_c(self):
+    def pick_staticmodbus_c(self):
+        directory = QtWidgets.QFileDialog.getExistingDirectory(parent=self)
+
+        if len(directory) == 0:
+            return
+
+        self.ui.staticmodbus_c.setText(directory)
+
+    def pick_sunspec_tables_c(self):
         filters = (
-            ("SunSpec Bitfields C", ["c"]),
+            ("SunSpec Tables C", ["c"]),
             all_files_filter,
         )
 
         self.file_dialog(
-            target=self.ui.sunspec_bitfields_c,
+            target=self.ui.sunspec_tables_c,
```

Comment:
is this right?  not sure what im looking at but it seems slightly off pattern

Reply:
Yes, it's right, but I understand the confusion. What was called "sunspec_bitfields_c" was changed to "bitfields_c" (which is a combination of SunSpec and static modbus bitfields). The section of code that refers to "sunspec_tables_c" has been swapped with the code that refers to "bitfields_c". So it's a name change plus a code relocation that is causing confusion in the diff.